src/javascript/app_2/Stores/Modules/Trading/trade-store.js

```diff
@@ -523,6 +531,25 @@ export default class TradeStore extends BaseStore {
         });
         this.updateQueryString();
         this.onSwitchAccount(this.accountSwitcherListener);
+        this.onLoadingMount();
+    }
+
+    @action.bound
+    onLoadingMount() {
+        setTimeout(() => {
+            this.updateLoadingStatus('Retrieving market symbols...');
+        });
+        setTimeout(() => {
+            this.updateLoadingStatus('');
```

Comment:
I don't think this is necessary?

Reply:
that is to make the status rerender henceforth the animation fade in coming